### DISCUSSION TITLE
Formating incorrect -format 'roundrectangle 1,1 ... unable to open image "1,1"

### DISCUSSION THREAD

**Discussion by fuzzy-andreas** [ImageMagick/ImageMagick#7731] [Help] [answered]
### ImageMagick version

7.1.1-Q16-HDRI

### Operating system, version and so on

Windows Server 2016

### Description

Hello,
I have a issue when executing 
magick thumbnail.gif -format 'roundrectangle 1,1 %[fx:w+4],%[fx:h+4] 15,15'... 
The answer in the command line is "magick: unable to open image '1,1': No such file or directory @ error/blob.c/OpenBlob/3569"
Same if i write 
-format 'roundrectangle 1.1 %[fx:w+4],%[fx:h+4] 15.15'... (dot instead of comma)

ok, its an absolute beginners question, i know :-)
But how i have to format the command correctly for Windows? Any help i much appreciated!
Its from https://usage.imagemagick.org/thumbnails/#rounded

Kind regards,
andreas

**Comment by dlemstra** [comment, marked as answer]
On windows you should use `"` instead of `'`.

**Reply by fuzzy-andreas** [reply]
Thats it! Thank you for the fast reply!
